tests/test_dataset_manager.py

```diff
@@ -132,6 +132,25 @@ def test_no_yaml_creation_repeat(good_dataset_manager):
     
     clean_up(good_dataset_manager)
 
+def test_ed_directory_format(good_dataset_manager):
+    good_dataset_manager.bootstrap()
+    ed_directory = good_dataset_manager._generate_ed_directory()
+    filepath = os.path.join(good_dataset_manager._raw_filepath, 'datasets.yaml')
+    mappings = good_dataset_manager._mappings
+
+    assert type(ed_directory == dict), \
+        "ED Directory is not a dictionary!"
+
+    for encoding, data_tuple in ed_directory.items():
+        dataset = data_tuple[0]
+        assert encoding in mappings.keys(), \
```

Comment:
So looks like we're just checking that every item in the encoding is in mappings.keys() shouldn't we also be checking that every item from mappings.keys() that we expect to be in the encoding is there?

Reply:
Nvm